google-account-plugin/src/com/google/cloud/tools/intellij/login/IntellijGoogleLoginService.java

```diff
@@ -621,14 +588,7 @@ public void initializeUsers() {
           continue;
         }
 
-        IGoogleLoginCompletedCallback callback = new IGoogleLoginCompletedCallback() {
-          @Override
-          public void onLoginCompleted() {
-            uiFacade.notifyStatusIndicator();
-          }
-        };
-
-        users.addUser(new CredentialedUser(delegate, callback));
+        users.addUser(new CredentialedUser(delegate, null));
```

Comment:
sometimes we add a comment next to unamed parameters to make things clearer

e.g. `new CredentialedUser(delegate, null /*variableNameFromSignature*/)`

This is a bit controversial, especially if you have IJ's new named parameter syntactic sugar thing on (I turned it off because I found it was buggy). But I think it helps readability.